sdk/eventhubs/azure-messaging-eventhubs/README.md

```diff
@@ -313,7 +313,6 @@ class SimplePartitionProcessor extends PartitionProcessor {
     /**
      * Processes the event data.
      */
-    @Override
```

Comment:
Instead of removing it, commenting it out won't cause any issues with javadoc. Change @Override to //@Override